druid/src/widget/container.rs

```diff
@@ -74,6 +74,11 @@ impl<T: Data> Container<T> {
         self.background = Some(brush.into());
     }
 
+    /// Removes background.
+    pub fn remove_background(&mut self) {
```

Comment:
I'm used to the 'set/clear' and 'add/remove' verbs used in pairs. This would be `set/remove`, so would the right verb rather be 'clear_background'?

Reply:
I agree. Made the change.